ethexe/common/src/mock.rs

```diff
@@ -167,13 +167,13 @@ pub struct PreparedBlockData {
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
-pub struct BlockData {
+pub struct BlockFullData {
```

Comment:
```suggestion
pub struct FullBlockData {
```
To be consistent with `SmallBlockData`